siliconcompiler/package/https.py

```
@@ -136,12 +137,12 @@ def resolve_remote(self) -> None:
         # Attempt to extract as a tarball, fall back to zip
         try:
             with tarfile.open(fileobj=fileobj, mode='r:gz') as tar_ref:
-                tar_ref.extractall(path=self.cache_path)
+                tar_ref.extractall(path=self.cache_path, **tar_extract_kwargs())
```

Ruff (0.15.21)

[error] Uses of `tarfile.extractall()`

(S202)